src/pretix/static/pretixcontrol/js/ui/main.js

```diff
@@ -699,6 +699,35 @@ var form_handlers = function (el) {
     el.find("input[name*=question], select[name*=question]").change(questions_toggle_dependent);
     questions_toggle_dependent();
     questions_init_photos(el);
+
+    var lastFocusedInput;
+    $(document).on('focusin', 'input, textarea', function(e) {
+        lastFocusedInput = e.target;
+    }).on("click", function(e) {
+        if (e.target.classList.contains('content-placeholder')) {
+            var container = e.target.parentNode.parentNode;
```

Comment:
$(e.target).closest(".form-group") might be more flexible, not sure

Reply:
Oh yes! I did not look that far up the DOM-tree for a closest-selector so chose the ugly parentNode.parentNode, but I initially hestitated to do so. Yours is way better!